examples/service-instance.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceInstance
+metadata:
+  name: skeleton-service-instance
+  namespace: test-ns
+spec:
+  clusterServiceClassExternalName: skeleton-example-service
+  clusterServicePlanExternalName: default
+  parameters:
+    param-1: value-1
```

Comment:
Maybe we could make this less vague, would be helpful to newcomers? Or add a comment above it?

Reply:
Agree, this was basically a lift-n-shift from service-catalog